cpp/src/Glacier2Lib/SessionHelper.cpp

```diff
@@ -450,20 +450,22 @@ SessionHelperI::connected(const Glacier2::RouterPrx& router, const optional<Glac
     //
     Ice::ConnectionPtr conn = router->ice_getCachedConnection();
     string category = router->getCategoryForClient();
-    int32_t acmTimeout = 0;
+    int32_t remoteIdleTimeout = 0;
     try
     {
-        acmTimeout = router->getACMTimeout();
+        remoteIdleTimeout = router->getACMTimeout();
     }
     catch (const Ice::OperationNotExistException&)
     {
     }
 
-    if (acmTimeout <= 0)
+    if (remoteIdleTimeout <= 0)
     {
-        acmTimeout = static_cast<int32_t>(router->getSessionTimeout());
+        remoteIdleTimeout = static_cast<int32_t>(router->getSessionTimeout());
     }
 
+    // TODO: verify remote idle timeout is compatible with local idle timeout
```

Comment:
Related question: what are compatible idle timeouts?
Do we take into account the setting for the local EnableIdleCheck too?